src/renderers/testing/ReactTestReconcileTransaction.js

```diff
@@ -57,9 +57,10 @@ var TRANSACTION_WRAPPERS = [ON_DOM_READY_QUEUEING];
  *
  * @class ReactTestReconcileTransaction
  */
-function ReactTestReconcileTransaction() {
+function ReactTestReconcileTransaction(mockConfig) {
+  this._mockConfig = mockConfig;
```

Comment:
Let’s keep it without underscore for consistency.
Also let’s move assignment just before `reactMountReady` assignment.